Website improvements

7elia opened the pull request:
## Proposed changes

I changed the padding and gaps of the nav elements that handle sorting. I also fixed an issue with text inputs being pushed down.

I also added a button to hide the "Under construction" bar that appears on the new website, and moved the bar itself under the global horizontal navbar.

Another thing I added is that the page where you manage your editors shows "No editors" when you have no editors or if you aren't an editor for any channels.

## Types of changes

What types of changes does your code introduce to 7TV?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/SevenTV/SevenTV/blob/main/CONTRIBUTING.md) doc
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged

lennartkloock commented:
@7elia why did you remove the top margin from the text input? what exactly was the issue with them being pushed down?

7elia commented:
1. I removed the top margin from the text input because it made it so the input fields weren't aligned with the center of its parent as you can see on these screenshots:
![image](https://github.com/user-attachments/assets/057d75cf-de6d-4ca4-b6cc-0cf3eb78b79e)
![image](https://github.com/user-attachments/assets/aa50da89-3dd3-47b8-b46e-052286b9cec2)
After my change it looks like this:
![image](https://github.com/user-attachments/assets/65347093-c1e7-4031-bd03-8a4fad77fea0)
![image](https://github.com/user-attachments/assets/86b31364-81b4-4896-a823-879494aeba34)
2. I added the padding and gap to the tab buttons because I felt it looked weird whenever another button was hovered while the one next to it was active:
![image](https://github.com/user-attachments/assets/81738b03-3d1f-49b2-a992-3aa07bb9d90e)
This happens because of the border radius combined with there being no gap between the buttons. So to fix it I added a gap, but then that looks weird because there's no space around it, so I added the same amount of padding to the entire element.
![image](https://github.com/user-attachments/assets/e60106a1-4e04-4fa1-9a3e-85278687b647)
I understand this change deviates heavily from the original design, but in my opinion, comparing these two images, the second one looks far better. Of course this `.3rem` value can be tweaked if you feel it is too much. It can also be removed if you want.

lennartkloock commented:
I reverted 1 and fixed the actual issue. The margin was there to add spacing to labels which are passed as children. But for the search inputs I pass absolute elements as children but it still added the margin.
